### DISCUSSION TITLE
Deleting permission

### DISCUSSION THREAD

**Discussion by neoacevedo** [spatie/laravel-permission#1871] [Q&A]
I try to delete a permission which isn't attached to any role, but when I run `$permission->delete();` it always returns false and, obviously the permission is never deleted from the database.

Is it supported by this package?

**Comment by erikn69** [comment]
>Is it supported by this package?

Yes, all methods from `\Illuminate\Database\Eloquent\Model` are available
https://github.com/spatie/laravel-permission/blob/2512d5a22aaae0bab8d62a02e502a603776da9da/src/Models/Permission.php#L16
https://github.com/spatie/laravel-permission/blob/2512d5a22aaae0bab8d62a02e502a603776da9da/src/Traits/HasRoles.php#L18-L27

**Reply by neoacevedo** [reply]
But, as I said, when I try to delete a permission, the delete method returns false.

**Reply by erikn69** [reply]
![image](https://user-images.githubusercontent.com/4933954/134406678-f3b56ae5-7549-4ca7-ba42-6aeebc791c07.png)

works for everyone, it's probably your implementation, i have 3 apps(Laravel 6 and 8) with this package, full CRUD
